src/pages/Home.js

```diff
@@ -1,4 +1,6 @@
 import React, { Component } from 'react';
+import { Row, Column } from 'react-foundation';
```

Comment:
Is there a specific reason why you want to use react-foundation instead of react-bootstrap? FYI, react-bootstrap also has <Row> and <Col>, and react-bootstrap is already installed as dependencies. If you really want to use react-foundation, please also add your package.json (not package-lock.json)